ampd/src/grpc/error.rs

```diff
@@ -115,6 +115,27 @@ impl From<&broadcaster_v2::Error> for Error {
     }
 }
 
+impl From<&cosmos::Error> for Error {
+    fn from(err: &cosmos::Error) -> Self {
+        match err {
+            cosmos::Error::GrpcConnection(_) | cosmos::Error::GrpcRequest(_) => {
```

Comment:
is the inner error not useful?

Reply:
They are logged elsewhere. I don't think they are useful to the client as we want to avoid blindly passing error from cosmos gRPC to ampd SDK.

Reply:
does the logging happen in the trace layer?

Reply:
The inner gets logged here https://github.com/axelarnetwork/axelar-amplifier/blob/26548a69b40cdfce4d988b382d3c7b10b9d21358/ampd/src/grpc/blockchain_service.rs#L90, and the Status we return in the end gets logged in the trace layer.